Buildings/HeatTransfer/Conduction/MultiLayer.mo

```diff
@@ -10,9 +10,22 @@ model MultiLayer
     "Heat flow rate from state i to i+1";
   extends Buildings.HeatTransfer.Conduction.BaseClasses.PartialConstruction;
 
+  parameter Boolean placeCapacityAtSurf_a=false
+    "Set to true to place the capacity at the surface a of the layer"
+    annotation (Dialog(tab="Dynamics"),
+                Evaluate=true);
+  parameter Boolean placeCapacityAtSurf_b=false
+    "Set to true to place the capacity at the surface b of the layer"
+    annotation (Dialog(tab="Dynamics"),
+                Evaluate=true);
+  parameter Integer nSta2[nLay]={layers.material[i].nSta for i in 1:nLay}
```

Comment:
Yes it needs to be public otherwise models which use `Buildings.Fluid.HeatExchangers.RadiantSlabs.SingleCircuitSlab` will have following translation warning

> Modifier for protected component nSta2. This is not legal Modelica.
> File: /home/thierry/vmWareLinux/proj/buildings_library/models/modelica/git/buildings/modelica-buildings/Buildings/Fluid/HeatExchangers/RadiantSlabs/SingleCircuitSlab.mo, line 39